cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -434,6 +434,31 @@ void FliImpl::get_sim_precision(int32_t *precision)
     *precision = mti_GetResolutionLimit();
 }
 
+const char *FliImpl::get_simulator_product()
+{
+    if (m_product.empty() && m_version.empty()) {
+        const char *c_info       = mti_GetProductVersion();      // Returned pointer must not be freed
+        const std::string info   = c_info;
+        const std::string search = " Version ";
+        const std::size_t found  = info.find(search);
```

Comment:
nit: could make this shorter
```suggestion
        const std::string info   = mti_GetProductVersion();
        const std::size_t found  = info.find(" Version ");
```

Reply:
`search` is used later, and I'd rather not hardcode the length, but I did fold `c_info` away.